database.js

```diff
@@ -32,15 +31,9 @@ module.exports = function(config) {
         PageUrl: pageUrl,
       };
 
-      _.forEach(clientFeatures, (value, key) => {
-        item[key] = value;
-      });
-      _.forEach(connectionFeatures, (value, key) => {
-        item[key] = value;
-      });
-      _.forEach(streamFeatures, (value, key) => {
-        item[key] = value;
-      });
+      Object.keys(clientFeatures).forEach(key => item[key] = clientFeatures[key]);
+      Object.keys(connectionFeatures).forEach(key => item[key] = connectionFeatures[key]);
+      Object.keys(streamFeatures).forEach(key => item[key] = streamFeatures[key]);
```

Comment:
💅 you may use `Object.assign` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign